app/routes.py

```diff
@@ -27,30 +27,13 @@ def change():
         return redirect(url_for('home'))
     return render_template('change.html', title='Edit Info', form=form)
 
-@app.route('/uk')
-def uk():
-    articles = Article.query.all()
-    return render_template('uk.html',articles=articles)
 
-@app.route('/uae')
-def uae():
-    articles = Article.query.all()
-    return render_template('uae.html',articles=articles)
 
-@app.route('/russia')
-def russia():
+@app.route('/<cntry>')
```

Comment:
One function that takes in parameter the country and passes it to the common layout page. This removes the need for separate functions for different countries. So, new countries can be added easily.